build/x86_64_sles12/Dockerfile

```diff
@@ -18,6 +18,9 @@ RUN zypper -n refresh \
  # RVM GPG keys.
  && mkdir -p ~/.gnupg && echo "disable-ipv6" >> ~/.gnupg/dirmngr.conf \
  && gpg2 --keyserver hkp://keyserver.ubuntu.com --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB \
+ # Remove expired root certificate.
+ && mv /var/lib/ca-certificates/pem/DST_Root_CA_X3.pem /etc/pki/trust/blacklist/ \
```

Comment:
This is still necessary even with the following `update` call?
Will this file definitely be present (bc otherwise it'll fail out)?

Reply:
The `update` call simply sprinkles the certificates from the source-of-truth (`trust` database) to various places on the filesystem where other (older) applications expect to find them. It does not actually contact any external update server. This command tells the `update` call to ignore the faulty certificate.
And yes, the file is present in the base image.

Reply:
👍 🙌